core/src/main/java/org/mvcspec/ozark/core/ViewResponseFilter.java

```diff
@@ -80,6 +81,7 @@
 public class ViewResponseFilter implements ContainerResponseFilter {
 
     private static final String REDIRECT = "redirect:";
+    private static final String DOT_CHAR = ".";
```

Comment:
Not sure how others feel about that, but defining a constant for the dot character isn't really required. I would be fine with inlining it.

Reply:
Fixed.